source/_docs/drupal-cron.md

```diff
@@ -57,14 +57,29 @@ Another very effective solution is to leverage a service such as [EasyCron](http
 
 
 ## Disable Cron
-Pantheon's cron execution cannot be disabled and will run every hour on all environments, even if cron is disabled in Drupal. Using a module like Elysia Cron will allow more granular control of what happens when Drupal cron runs.
+Pantheon's cron execution cannot be disabled and will run every hour on all environments, even if cron is disabled in Drupal. 
```

Comment:
I'm having a hard time understanding how this sentence isn't being contradicted by Lines 62 and 68. Please Clarify.

Reply:
Agreed. We need clarification on whether disabling cron globally using Elysia affects platform cron execution. If it _can_ disable it, we can remove this line. Otherwise, we need to keep this line and clarify what specifically gets disabled.